spec/bifrost/topic_spec.rb

```diff
@@ -88,10 +87,24 @@
       expect(new_topic.add_subscriber(subscriber)).to be_truthy
       expect(new_topic.add_subscriber(subscriber)).to be_falsey
     end
-  end
 
-  it 'should not accept names with spaces in them' do
-    invalid_topic = Bifrost::Topic.new('topic name')
-    expect { invalid_topic.save }.to raise_error(URI::InvalidURIError)
+    context 'with no subscribers' do
+      it 'should return no subscribers' do
+        expect(new_topic.subscribers).to be_empty
+      end
+    end
+
+    context 'with subscribers' do
+      before(:each) do
```

Comment:
@lfportal at the parent level are we clearing all the topics at the end of the specs?